### PR TITLE
#284: Created new flavor EXASOL-7.1.0-without-python2.7.

### LINKED ISSUE #284: Split Python2 support
## Background

Python2 is deprecated and no further fixes (security fixes!!) are provided. 
Need to remove Python2 in script languages in order to avoid possible issues.

## Acceptance Criteria

Remove Python2 for standadard container and create a separate container for Python2.

### PR DESCRIPTION
**Pull request by tomuben**
Closes #284

### CODE REVIEW

**Comment by tkilias** [comment]
Hm, I am thinking about, if we should keep the naming you used, or switch the name and call the container with python2 the normal standard flavor and the other one standard-without-python2.

File: flavors/README.md [hunk 1]
```diff
@@ -17,10 +17,18 @@ It provides the languages Java, Python 2/3 and R.
...
+  - Base Image: ubuntu:18.04
+  - Compatible Exasol Versions: 7.1.\* (but likely also with 7.0 and 6.2.\*) 
+  - Available languages: Java 11, Python 3.6 and R 3.4
+  - [Here](standard-EXASOL-7.1.0-without-python2.7/FLAVOR_DESCRIPTION.md) you can find more details about the flavor, such as the provided packages.
+
 - standard-EXASOL-7.1.0
   - Base Image: ubuntu:18.04
   - Compatible Exasol Versions: 7.1.\* (but likely also with 7.0 and 6.2.\*) 
-  - Available languages: Java 11, Python 2.7, Python 3.6 and R 3.4
+  - Available languages: Java 11, Python2.7, Python 3.6 and R 3.4
```
Comment (tkilias) [inline flavors/README.md:31]:
```suggestion
  - Available languages: Java 11, Python 2.7, Python 3.6 and R 3.4
```

File: flavors/standard-EXASOL-7.1.0-without-python2.7/flavor_base/build_deps/packages/apt_get_packages [hunk 2]
```diff
@@ -0,0 +1,11 @@
+software-properties-common
```
Comment (tkilias) [inline flavors/standard-EXASOL-7.1.0-without-python2.7/flavor_base/build_deps/packages/apt_get_packages:1]: Here we should create a ticket to pin this dependencies in the future.

**Review by tkilias — COMMENTED** [review]
Nothing major, see my comments

**Comment by tkilias** [comment]
Maybe, it is wise to add also a test which check if the python2 binary and the shared lib are not anymore in the container.

File: flavors/standard-EXASOL-7.1.0-without-python2.7/flavor_base/build_deps/packages/apt_get_packages (hunk 2 above)
Comment (tomuben) [reply flavors/standard-EXASOL-7.1.0-without-python2.7/flavor_base/build_deps/packages/apt_get_packages:1]: Created #292

**Review by tkilias — APPROVED** [review]

**Pull request merged**